ansible_collections/arista/avd/roles/eos_designs/templates/network_services/port-channel-interfaces.j2

```diff
@@ -4,7 +4,7 @@
 port_channel_interfaces:
 {% set point_to_point_service_subif_parents = [] %}
 {# pseudowire interfaces #}
-{% for tenant in network_services_data.tenants | arista.avd.natural_sort('name') %}
+{% for tenant in network_services_data.tenants | arista.avd.convert_dicts('name') | arista.avd.natural_sort('name') %}
```

Comment:
I believe all this is not needed for `network_services_data`. The input variables are read in the logic.j2 and written out to this model which is purely internal.

Reply:
ohh okay, I was not getting where is network_services_data. So here we need not to change templates, will keep only md file in this PR